scalableminds/webknossos
# Some cleanup in annotation sql queries

Some convenience methods have been created in the meantime, time to use them

### URL of deployed dev instance (used for testing):
- https://sqlcleanupannotation.webknossos.xyz

### Steps to test:
- create explorative, edit its visibility, name, description
- create tasks, request, trace finish
- should be listed properly in task admin view and project progress overview

------
